src/applications/personalization/view-dependents/layouts/ViewDependentsLayout.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+
+import Breadcrumbs from '@department-of-veterans-affairs/formation-react/Breadcrumbs';
+import ViewDependentsLists from './ViewDependentsLists';
+import ViewDependentsSidebar from '../components/ViewDependentsSidebar/ViewDependentsSidebar';
+import ViewDependentsHeader from '../components/ViewDependentsHeader/ViewDependentsHeader';
+
+class ViewDependentsLayout extends Component {
+  render() {
+    const breadcrumbLinks = [
+      <a href="/" aria-label="back to VA Home page" key="1">
+        Home
+      </a>,
+      <a
+        href="/disability"
+        aria-label="Back to the Disability Benefits page"
+        key="2"
+      >
+        Disability Benefits
+      </a>,
+      <a
+        href="/disability/add-remove-dependent"
+        aria-label="Back to the Add or remove dependents page"
+        key="3"
+      >
+        Add or remove dependents
+      </a>,
+      <a href="/disability/view-dependents/" key="4">
```

Comment:
I just noticed this on our Rated Disabilities page too, do these last breadcrumb links need aria labels?

Reply:
Im honestly not sure. I took the code directly from the pattern library but that might not be the most up to date reference.